dpks/classification.py

```diff
@@ -70,7 +64,12 @@ def predict(self, X):
     def cross_validation(self, X, y, k_folds: int = 5):
         self.scores = cross_val_score(self.classifier, X, y, cv=k_folds)
 
-    def interpret(self, X):
+    def interpret(self, X, shap_algorithm="auto"):
+        self.X = X
+
+        if not self.shap_algorithm:
+            self.shap_algorithm = shap_algorithm
```

Comment:
Is there any reason to have this as an attribute if it is now just set to ```None``` by default and only used in this ```interpret()``` function?